modules/main.js

```diff
@@ -186,17 +203,17 @@ function processAction(aWindow, aAction)
   var value = actions[2];
   switch (action) {
   case 'accept':
-    doc.documentElement.acceptDialog();
+    commonDialog.onButton0();
```

Comment:
`commonDialog.onButton0()` doesn't work without tabmodaldialog on my environment.
I think it should be replaced with `commonDialog.ui.button0.click()`.